sql/core/src/test/gen-java/org/apache/spark/sql/execution/datasources/parquet/test/avro/AvroArrayOfArray.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Autogenerated by Avro
```

Comment:
We can merge this so its included in 1.5, but why are we checking in auto generated code?  Can't we build it on demand.